helix-view/src/info.rs

```diff
@@ -16,33 +16,55 @@ pub struct Info {
 }
 
 impl Info {
-    pub fn new(title: &str, body: Vec<(&str, BTreeSet<KeyEvent>)>) -> Info {
-        let body = body
-            .into_iter()
-            .map(|(desc, events)| {
-                let events = events.iter().map(ToString::to_string).collect::<Vec<_>>();
-                (desc, events.join(", "))
-            })
-            .collect::<Vec<_>>();
+    pub fn new(title: &str, body: Vec<(String, String)>) -> Self {
+        if body.is_empty() {
+            return Self {
+                title: title.to_string(),
+                height: 1,
+                width: title.len() as u16,
+                text: "".to_string(),
+            };
+        }
 
-        let keymaps_width = body.iter().map(|r| r.1.len()).max().unwrap();
+        let item_width = body.iter().map(|(item, _)| item.len()).max().unwrap();
```

Comment:
This will cause an issue with CJK characters if we get width from user input, if say user copied some double-width characters. Can try with this
```
お前はもう死んでいる
```

Reply:
Fixed.